### PR TITLE
Use newest version of setup script for example

### PR DESCRIPTION
**Pull request by gregorybleiker**
The foxx service creation class creates a setup script skeleton. The setup script for this sample should use the same structure.

### CODE REVIEW

**Comment by pluma** [comment]
The warnings the generator adds are actually a bad idea because the setup is supposed to be reentrant and some parts of Foxx will run the setup script multiple times (e.g. during development mode) leading to unnecessary noise.

We should probably fix the generator, not the example.

File: Documentation/Books/Manual/Foxx/Users.md [hunk 1]
```diff
@@ -27,18 +27,30 @@ Then create the setup script with the following content:
...
-  db._createDocumentCollection(sessions);
+const documentCollections = ['sessions','users'];
+const edgeCollections = [];
+
+for (const localName of documentCollections) {
+  const qualifiedName = module.context.collectionName(localName);
+  if (!db._collection(qualifiedName)) {
+    db._createDocumentCollection(qualifiedName);
+  } else if (module.context.isProduction) {
+    console.warn(`collection ${qualifiedName} already exists. Leaving it untouched.`)
```
Comment (pluma) [inline Documentation/Books/Manual/Foxx/Users.md]: The generated setups actually use `console.debug` (which isn't logged by default so it results in less noise). So if you want to reflect the generator output, you should probably use that.

File: Documentation/Books/Manual/Foxx/Users.md [hunk 2]
```diff
@@ -27,18 +27,30 @@ Then create the setup script with the following content:
 ```js
 'use strict';
 const db = require('@arangodb').db;
-const sessions = module.context.collectionName('sessions');
-const users = module.context.collectionName('users');
-
-if (!db._collection(sessions)) {
-  db._createDocumentCollection(sessions);
+const documentCollections = ['sessions','users'];
```
Comment (pluma) [inline Documentation/Books/Manual/Foxx/Users.md]: Nitpick: there's a space missing after the comma.

File: Documentation/Books/Manual/Foxx/Users.md [hunk 3]
```diff
@@ -27,18 +27,30 @@ Then create the setup script with the following content:
...
 }
 
-if (!db._collection(users)) {
-  db._createDocumentCollection(users);
+for (const localName of edgeCollections) {
+  const qualifiedName = module.context.collectionName(localName);
+  if (!db._collection(qualifiedName)) {
+    db._createEdgeCollection(qualifiedName);
+  } else if (module.context.isProduction) {
+    console.warn(`collection ${qualifiedName} already exists. Leaving it untouched.`)
```
Comment (pluma) [inline Documentation/Books/Manual/Foxx/Users.md]: See above, this should be `console.debug`.

File: Documentation/Books/Manual/Foxx/Users.md [hunk 4]
```diff
@@ -27,18 +27,30 @@ Then create the setup script with the following content:
...
+  }
 }
 
-db._collection(users).ensureIndex({
+const qualifiedUsers = module.context.collectionName('users');
+
+db._collection(qualifiedUsers).ensureIndex({
```
Comment (pluma) [inline Documentation/Books/Manual/Foxx/Users.md]: These three lines can be simplified to:

```js
module.context.collection('users').ensureIndex({
```

**Comment by pluma** [comment]
Thank you for the PR. For legal reasons we need a signed Contributor License Agreement (CLA) before we can merge your code into the project.

You can download the form from our website: https://www.arangodb.com/documents/cla.pdf

Once you've filled out and signed it, please send it to us via e-mail: cla@arangodb.com or fax: +49-221-2722999-88.

**Comment by fceller** [comment]
@pluma CLA is available, can we merge?

**Comment by pluma** [comment]
@fceller Once the requested changes have been made, the PR can be merged.

**Comment by pluma** [comment]
@gregorybleiker Thanks again for your contribution. However I can't merge this without the requested changes. Can you take another look at this PR?

**Comment by fceller** [comment]
@pluma what is status of this PR?

**Comment by pluma** [comment]
@gregorybleiker I'm closing this PR as it has been inactive for over a month. Feel free to reopen this PR or open a new PR at any time if you can make the requested changes.

**Comment by gregorybleiker** [comment]
Oh, should I have done something?

**Pull request closed without merging**